### PR TITLE
Add a delay option for untriggering the sensor

### PR DESCRIPTION
**Pull request by mporcheron**
This PR adds a `delayOff` option, set to 0 by default, to delay the occupancy sensor being 'untriggered' when playback is stopped. The timeout is cancelled if another play event occurs before the untriggering occurs.

### CODE REVIEW

File: index.js
Comment (alexjsp) [inline index.js:149]: Thanks for the indentation fix. 💜

File: index.js
Comment (alexjsp) [inline index.js]: I don't love that this creates an unnecessary 0 second timeout, and creates duplicated log events, for people with no delay set.

Would you consider moving this functionality around so that a timeout is only set (and the additional `scheduled` log message only shown) when `this.delayOff > 0`?

**Review by alexjsp — CHANGES_REQUESTED** [review]
Great work, love this addition. 💜 

One thing I'd love you to tweak before I merge it if possible though?

File: index.js
Comment (mporcheron) [reply index.js:149]: Truthfully, a plugin did it without me realising :-)

**Comment by mporcheron** [comment]
I've done that (with a typo in the commit message, ah well).

**Review by alexjsp — CHANGES_REQUESTED** [review]
Hmm, sorry for not spotting this before but I’ve just realised the timeout isn’t stored on the sensor so it’s global for all sensors, but that means if a second sensor stops playing while the timeout is active for the first that the first sensor will never turn off, right? Unless I’m missing something. 🤔

**Comment by mporcheron** [comment]
Serves me right for throwing this together without thinking it through. Moved it into the per-sensor config. I think this should do it — I've some some testing, but only with a single sensor. If I get more time, I'll test with multiple sensors.

I'd hold off merging just yet.

**Review by alexjsp — APPROVED** [review]
I did some testing and this works great now, going to merge into `master` and push a new version.

Thanks for the hard work! 💜

**Comment by mporcheron** [comment]
Sorry for not getting to testing it — it was on my (ever growing) to-do list. Glad it works, thanks for making this plugin in the first place!

**Pull request merged**
